rosdep/python.yaml

```diff
@@ -7592,6 +7592,29 @@ python3-rospkg-modules:
     pip:
       packages: [rospkg]
   ubuntu: [python3-rospkg-modules]
+python3-rpi.gpio:
+  debian:
+    bullseye: [python3-rpi.gpio]
+    buster: [python3-rpi.gpio]
+    jessie:
+      pip:
+        packages: [RPi.GPIO]
+    stretch:
+      pip:
+        packages: [RPi.GPIO]
+  fedora: [python3-RPi.GPIO]
```

Comment:
could you provide a link?
I couldn't find this fedora package.

P.S.: When opening a PR to add a rosdep entry, please complete the `Links to Distribution Packages` section.